packages/node_modules/@webex/plugin-meetings/src/common/errors/webex-errors.js

```diff
@@ -0,0 +1,95 @@
+// The document would auto generate the doc for errors
+
+// 1) Error codes for Oauth, login, mercury should be separated out (Code range)
+// 2) Errors from the actual locus server or other server in case, we can use the same error code as locus and redirect it
+// 3) Any errors we generate from the SDK can be split into various categories
+// Parameter validation, user action, connection, media specific(They can have different range)
+
+
+// TODO: all the errors from the server need to be captured
+
+// add a way to log and send metrics if needed
+const WebexMeetingsError = require('./webex-meetings-error');
+
+const WebExMeetingsErrors = {};
+
+/**
+ * Create a {@link WebexMeetingsError} for a given code and message.
+ * @private
+ * @param {number} [code] - Error code
+ * @param {string} [message] - Error message
+ * @returns {WebexMeetingsError}
+ */
+exports.createMeetingsError = function createMeetingsError(code, message) {
+  code = typeof code === 'number' ? code : 0;
+  message = typeof message === 'string' && message ? message : 'Unknown error';
+
+  return WebExMeetingsErrors[code] ? new WebExMeetingsErrors[code]() : new WebexMeetingsError(code, message);
+};
+
+/**
+ * @class MeetingNotActiveError
+ * @classdesc Raised whenever Meeting has already ended and user tries to do a action.
+ * @extends WebexMeetingsError
+ * @property {number} code - 30101
+ * @property {string} message - 'Meeting has already Ended or not Active'
+ */
+class MeetingNotActiveError extends WebexMeetingsError {
+  constructor() {
+    super(30101, 'Meeting has already Ended or not Active');
+  }
+}
+
+exports.MeetingNotActiveError = MeetingNotActiveError;
+Object.defineProperty(WebExMeetingsErrors, 30101, {value: MeetingNotActiveError});
```

Comment:
```suggestion
WebExMeetingsErrors[MeetingNotActiveError.CODE] = MeetingNotActiveError;
```
This one you don't need to do.